ami/ml/post_processing/admin/small_size_filter_form.py

```
@@ -0,0 +1,28 @@
+
+class SmallSizeFilterActionForm(BasePostProcessingActionForm):
+    """Knobs surfaced when an admin triggers Small Size Filter."""
+
+    size_threshold = forms.FloatField(
+        label="Size threshold",
+        initial=SmallSizeFilterConfig.__fields__["size_threshold"].default,
+        min_value=0.0,
+        max_value=1.0,
+        help_text=(
+            "Minimum bounding-box area as a fraction of the source image area "
+            "(width × height). Detections smaller than this are flagged as "
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)